services/director-v2/src/simcore_service_director_v2/api/routes/dynamic_services.py

```diff
@@ -40,3 +70,165 @@ async def service_retrieve_data_on_ports(
     )
     # validate and return
     return RetrieveDataOutEnveloped.parse_obj(resp.json())
+
+
+@router.post(
+    "/{node_uuid}:start",
+    summary="start the dynamic-sidecar for this service",
+    responses={
+        status.HTTP_500_INTERNAL_SERVER_ERROR: {
+            "description": "Error while starting dynamic sidecar"
+        }
+    },
+)
+async def start_dynamic_sidecar(
+    node_uuid: UUID, model: StartDynamicSidecarModel, app: FastAPI = Depends(get_app)
+) -> Dict[str, str]:
+    debug_message = (
+        f"DYNAMIC_SIDECAR: user_id={model.user_id}, project_id={model.project_id}, "
+        f"service_key={model.service_key}, service_tag={model.service_tag}, "
+        f"node_uuid={node_uuid}"
+    )
+    log.debug(debug_message)
+
+    dynamic_sidecar_settings: DynamicSidecarSettings = get_settings(app)
```

Comment:
MINOR: Please use dependences for these things

Reply:
yes, also replaced get_monitor(app) with the dependency injection version.